src/components/PopOver/index.js

```diff
@@ -2,17 +2,19 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 import { CSSTransition } from "react-transition-group";
-import { themes, constants, spacing } from "../../theme";
-import { popContainersBoxShadow } from "../../theme/constants";
+import { themes, constants, spacing, zIndex } from "../../theme";
+import { popContainersSharpBoxShadow } from "../../theme/constants";
 
 const StyledPopOver = styled.div`
   background-color: ${themes.global.white.base};
-  border: 1px solid ${themes.global.gray02};
+  border: ${({ noBorders }) =>
+    !noBorders ? `1px solid ${themes.global.gray02}` : "0"};
```

Comment:
You could omit the `border` style attribute when the `noBorders` prop equals true.

E.g. 
```
${({ noBorders }) => !noBorders && `border:  1px solid ${themes.global.gray02}`};
```

Reply:
Done 🙂